packages/expo-video/ios/VideoView.swift

```diff
@@ -136,10 +161,47 @@ public final class VideoView: ExpoView, AVPlayerViewControllerDelegate {
   }
 
   public func playerViewControllerDidStartPictureInPicture(_ playerViewController: AVPlayerViewController) {
+    isInPictureInPicture = true
     onPictureInPictureStart()
   }
 
   public func playerViewControllerDidStopPictureInPicture(_ playerViewController: AVPlayerViewController) {
+    isInPictureInPicture = false
     onPictureInPictureStop()
   }
+
+  // MARK: - Utils
+
+  /**
+   * iOS automatically pauses videos when the app enters the background. Only way of avoiding this is to detach the player from the playerLayer.
```

Comment:
```suggestion
   * iOS automatically pauses videos when the app enters the background. Only way to avoid this is to detach the player from the playerLayer.
```